modules/ROOT/pages/managing-users.adoc

```diff
@@ -1,60 +1,74 @@
-= To Configure SAML
+= Configure SAML for SSO
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-. Make sure to configure your SAML provider setting up your Anypoint Platform organization as your audience.
-. Set the assertion consumer service to send a POST request to +https://anypoint.mulesoft.com/accounts/login/receive-id+.
-. Log into the master Organization in Anypoint Platform as Administrator.
-. In Anypoint Platform, click Access Management > External Identity.
-. From Identity Management, select SAML 2.0.
+Configure SAML to provide external authentication of users and Single Sign-on (SSO) capability.
+
+== Prerequisites
+
+Configure your SAML provider:
+
+* Set up your Anypoint Platform organization as your audience.
+* Set the assertion consumer service to send a `POST` request to `+https://anypoint.mulesoft.com/accounts/login/receive-id+`.
+
+== Configure SAML
+
+. In Anypoint Platform, sign into the master organization as an organization administrator.
+. In *Management Center*, click *Access Management*.
+. In the menu on the left, click *External Identity*.
+. Select *Identity Management > SAML 2.0*.
 +
 The External Identity - Identity Management SAML 2.0 form appears.
 +
-. Fill in the following required fields:
+. Fill in the required fields:
 +
-* Sign On URL
+* *Sign On URL* 
 +
 This redirect URL provided by the IdP for sign in. For example, `+https://example.com/sso/saml+`.
 +
-* Sign Off URL
+* *Sign Off URL*
 +
 URL to redirect sign out requests, so users both sign out of the Anypoint Platform and have their SAML user's status set to signed out.
```

Comment:
sign out requests > signout requests
, so > , so that  (but does your audience really need to know why?)

Reply:
Yes, they need to know why.